UMN-AutoPackager/public/Set-UMNGlobalConfig.ps1

```diff
@@ -0,0 +1,160 @@
+function Set-UMNGlobalConfig {
+    <#
+    .SYNOPSIS
+        Sets a value(s) for the global configuration JSON file for the auto packager.
+    .DESCRIPTION
+        This command sets in a json file a value for one of various global configurations used as part of the AutoPackager. It requires a the full path and name of the JSON file, the key to set, and the value to set it to.
+    .EXAMPLE
+        Set-UMNGlobalConfig -Path .\GlobalConfig.json -Key CompanyName  -Value "University of Minnesota"
+        Sets the value of CompanyName to University of Minnesota for the config.json file
+    .EXAMPLE
+        Set-UMNGlobalConfig -Path .\GlobalConfig.json -Key ConfigMgr -SiteServer "my.config.site"  -SiteCode "COM" -DownloadLocationPath "C:\\Temp" -ApplicationContentPath "\\appstorage.somewhere\"
+        Sets the values for a ConfigMgr site to be added to the config.json file
+    .EXAMPLE
+        Set-UMNGlobalConfig -Path .\GlobalConfig.json -Key RecipeLocations -Value "\\file.server\recipes" -LocationType "directory"
+        Adds the location "\\files.server\recipes" and LocationType "directory" to the RecipeLocations key in the JSON specified
+    .PARAMETER Path
+        The full path and file name of the JSON file to be updated
+    .PARAMETER Key
+        The key that you want to update in the Global Configuation JSON file
+    .PARAMETER Value
+        The single value you want to set the key to
+    .PARAMETER SiteServer
+        The ConfigMgr site server address
+    .PARAMETER SiteCode
+        The site code for this instance of ConfigMgr
+    .PARAMETER DownloadLocationPath
+        The full path where downloaded data will be temporarily stored
+    .PARAMETER ApplicationContentPath
+        The full path for where ConfigMgr applcation content will be stored
+    .PARAMETER LocationType
+        The location type for where the recipes will be stored for the autopackager. Directory will be the most common.
+    #>
```

Comment:
We should standardize around help being outside the function right above the delcaration.